bihire/property_pro_lite
# user can get product by product id

#### What does this PR do?
- fetches a property by its id
- get all the info related to the property

#### Description of Task to be completed
- A user should be able to fetch all information related to the property by using its id

#### How should this be manually tested?
- clone the repo and switch to the branch server
- start the server
- use the url `http://localhost:8081` 
- use postman to hit endpoints
- send url `http://localhost:8081/product/:productId`
- if successfull should receive status 200 as a response 
- if fail should receive status 404 as a response

#### Any background context you want to provide?
- N/A

#### What are the relevant pivotal tracker stories?
[#167059184](https://www.pivotaltracker.com/n/projects/2360884)
